api/src/Responder/Response/ApiResponse.php

```diff
@@ -187,4 +188,32 @@ protected function encode($data)
         $prettyPrint = $debug?JSON_PRETTY_PRINT:0;
         return json_encode($data, $prettyPrint);
     }
+
+    /**
+     * Creates a redirect response.
+     *
+     * @param  string  $url
+     * @param  int     $status
+     * @param  array   $headers
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return $this
+     */
+    public function redirect($url, $status = 302, array $headers = [])
+    {
+        if (empty($url)) {
+            throw new InvalidArgumentException('Cannot redirect to an empty URL.');
+        }
+
+        $this->setStatusCode($status);
+        $this->headers->add($headers);
+        $this->headers->set('Location', $url);
+
+        if (!$this->isRedirect()) {
+            throw new InvalidArgumentException(sprintf('The HTTP status code is not a redirect ("%s" given).', $status));
+        }
+
+        return $this;
```

Comment:
This method should not be chainable (it should be the end of the chain) as the `$this->isRedirect()` check will not fail if a `->setStatusCode()` is chained after this method. If this method returns void it cannot be mistakenly called at the wrong point in the chain.